live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/mapper/property/PropertiesMapper.java

```diff
@@ -57,31 +57,31 @@
 @RequiredArgsConstructor
 public class PropertiesMapper
 {
-    public Set<CustomPropertyDelta<?>> calculateCustomPropertiesDelta(RepoEvent<DataAttributes<NodeResource>> event)
+    public Set<PropertyDelta<?>> mapProperties(RepoEvent<DataAttributes<NodeResource>> event)
     {
         if (isEventTypeCreated(event))
         {
-            return allCustomPropertiesUpdated(event);
+            return allPropertiesUpdated(event);
         }
 
-        return someCustomPropertiesUpdated(event);
+        return somePropertiesUpdated(event);
     }
 
-    private Set<CustomPropertyDelta<?>> allCustomPropertiesUpdated(RepoEvent<DataAttributes<NodeResource>> event)
+    private Set<PropertyDelta<?>> allPropertiesUpdated(RepoEvent<DataAttributes<NodeResource>> event)
     {
-        Stream<CustomPropertyDelta<?>> propertyDeltas = customPropertiesStream(event.getData().getResource())
+        Stream<PropertyDelta<?>> propertyDeltas = streamProperties(event.getData().getResource())
                 .filter(property -> Objects.nonNull(property.getValue()))
-                .map(property -> CustomPropertyDelta.updated(property.getKey(), property.getValue()));
+                .map(property -> PropertyDelta.updated(property.getKey(), property.getValue()));
         return streamOfAllProperties(event, propertyDeltas);
     }
 
-    private Set<CustomPropertyDelta<?>> someCustomPropertiesUpdated(RepoEvent<DataAttributes<NodeResource>> event)
+    private Set<PropertyDelta<?>> somePropertiesUpdated(RepoEvent<DataAttributes<NodeResource>> event)
     {
-        Stream<CustomPropertyDelta<?>> propertyDeltas = calculatePropertiesDelta(event);
+        Stream<PropertyDelta<?>> propertyDeltas = calculatePropertiesDelta(event);
         return streamOfAllProperties(event, propertyDeltas);
     }
 
-    private Set<CustomPropertyDelta<?>> streamOfAllProperties(RepoEvent<DataAttributes<NodeResource>> event, Stream<CustomPropertyDelta<?>> propertyDeltas)
+    private Set<PropertyDelta<?>> streamOfAllProperties(RepoEvent<DataAttributes<NodeResource>> event, Stream<PropertyDelta<?>> propertyDeltas)
```

Comment:
I know that the property name change isn't hitting this - but the name of this method is a bit confusing as it returns a `Set` not a `Stream`.